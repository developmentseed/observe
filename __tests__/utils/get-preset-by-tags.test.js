/* global fetch */
/* global test */
/* global __dirname */
/* global expect */

import getPresetByTags from '../../app/utils/get-preset-by-tags'

test('get preset by tags', () => {
  const preset = getPresetByTags({ building: '*' })
  expect(preset).toMatchSnapshot()
})
