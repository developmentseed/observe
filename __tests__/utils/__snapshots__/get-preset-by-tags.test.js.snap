// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`get preset by tags 1`] = `
Object {
  "fields": Array [
    "name",
    "building",
    "levels",
    "height",
    "address",
  ],
  "geometry": Array [
    "point",
    "area",
  ],
  "icon": "maki-home",
  "matchScore": 0.6,
  "moreFields": Array [
    "architect",
    "building/material",
    "layer",
    "roof/colour",
    "smoking",
    "wheelchair",
  ],
  "name": "Building",
  "tags": Object {
    "building": "*",
  },
  "terms": Array [],
}
`;
